packages/embark-profiler/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "embark-profiler",
+  "version": "4.0.0",
+  "author": "Iuri Matias <iuri.matias@gmail.com>",
+  "contributors": [],
+  "description": "Smart contract documentation generator for Embark DApps",
```

Comment:
This is the description for graph, should be updated to something appropriate for profiler.